src/core.c/Rakudo/Internals/PostcircumfixAdverbs.pm6

```diff
@@ -296,7 +296,12 @@ augment class Rakudo::Internals {
         );
 
         # Perform the actual lookup and handling
+#?if moar
         my int $index = nqp::atpos_i(@pc-adverb-mapper,$bitmap);
+#?endif
+#?if !moar
+        my int $index = nqp::atpos(@pc-adverb-mapper,$bitmap);
```

Comment:
If I'm not mistaken ```@pc-adverb-mapper``` comes out as an Array on the JVM backend. So using ```nqp::atpos``` does not work. But if this line and line 411 are changed to 
```
my int $index = @pc-adverb-mapper[$bitmap];
```
then things seem to look better.

Reply:
Fixed with https://github.com/rakudo/rakudo/commit/76d3e33811 I hope.  Thanks for the nudges!

Reply:
It turned out that this wasn't enough. I had to remove the ```ìnt``` from the above line, too: https://github.com/rakudo/rakudo/pull/4163

But now the JVM backend seems to be fine (no additional spectest failures).